### ISSUE TITLE
F2 Always seems to load Oculus instead of OpenVR

### ISSUE THREAD

**Issue by andybak** [BananaHemic/gamelodge_unity#1]
ToggleVRMode() in VRSDKUtils has the following logic:

            if(_prevSDK == SDK.OpenVR)
            {
                newSDK = SDK.OpenVR;
                UnityEngine.XR.XRSettings.LoadDeviceByName("openvr");
            }
            else
            {
                newSDK = SDK.Oculus;
                UnityEngine.XR.XRSettings.LoadDeviceByName("oculus");
            }

So - we seem to only ever load openvr if it's already been loaded before? But how does it get loaded in the first place?

**Comment by Bananahavoc** [comment]
When launched from Steam you have the option to run Gamelodge in Desktop, Oculus, or OpenVR mode. This corresponds to a launch command of vrmode=oculus or vrmode=openvr. We might want to add some shortcuts so that you have the same set of options with Itch

**Comment by andybak** [comment]
I was running within Unity. What's the simplest way to simulate those command line switches in the editor?

**Comment by andybak** [comment]
(Might be worth adding this info to the repo readme)

**Comment by Bananahavoc** [comment]
Yea, I should add this info somewhere.
To simulate this in Unity, you'll want to check out the XR settings under project settings->Player. Just drag to the top the VR SDK that you want to test with

**Comment by andybak** [comment]
> Just drag to the top the VR SDK that you want to test with

Hmmmm. I thought I'd tried that. It seemed to default to "none" and I had to edit the source. Maybe it was user error...

**Comment by Bananahavoc** [comment]
Yes, "none" is desktop mode. Lmk if dragging Oculus/OpenVR to the top doesn't do what you expect
